### PR TITLE
Release notes for Android core 2.1.2

### PR DESCRIPTION
**Pull request by praveek**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/documentation/mobile-core/release-notes.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Release Notes
 
+## April 14, 2023
+
+### Android Mobile Core 2.1.2
+
+* Fixed an issue that prevented some in-app messaging show frequency rules from working correctly.
```
Comment (calebk1) [inline src/pages/documentation/mobile-core/release-notes.md:7]: What exactly do we mean here by "Show frequency rules?"

Is it like the IAM frequency rules weren't working properly?

**Review by calebk1 — COMMENTED** [review]
One question.

File: src/pages/documentation/mobile-core/release-notes.md (hunk 1 above)
Comment (praveek) [reply src/pages/documentation/mobile-core/release-notes.md:7]: These are rules which are configured to be shown once or on every launch.

Comment (praveek) [reply src/pages/documentation/mobile-core/release-notes.md:7]: @sbenedicadb Pinging you for the correct name.

Comment (sbenedicadb) [reply src/pages/documentation/mobile-core/release-notes.md:7]: @calebk1 that's correct.  show frequency is a section in part of the IAM publishing process where the marketeer determines how often this message will be shown.  can include values like "only once", "x number of times", "until y event happens", or simply "show it every time the conditions are met".

**Review by sbenedicadb — APPROVED** [review]

**Pull request merged**